lib/gear_application/gear_application.ex

```diff
@@ -29,7 +28,7 @@ defmodule Antikythera.GearApplication do
   alias AntikytheraCore.GearLog.Writer
 
   @doc false
-  defun start(gear_name :: v[GearName.t], children :: [Spec.spec]) :: {:ok, pid} do
+  defun start(gear_name :: v[GearName.t], children :: list) :: {:ok, pid} do
```

Comment:
Typespec here can be improved.
This `children` goes directly into [Supervisor.start_link/2](https://hexdocs.pm/elixir/1.5.3/Supervisor.html#start_link/2), in that the spec for `children` is `[:supervisor.child_spec | {module, term} | module]`.
(Notice the above link is for Elixir 1.5.3)

It is a bit verbose so let us put:
```ex
@type child_spec :: [:supervisor.child_spec | {module, term} | module]
```
and use it from other places including updated gear_template.

Reply:
fixed in 44fa02c